Libs/MRML/DisplayableManager/vtkMRMLModelSliceDisplayableManager.cxx

```diff
@@ -516,6 +516,18 @@ ::UpdateDisplayNodePipeline(vtkMRMLDisplayNode* displayNode, const Pipeline* pip
 #endif
     pipeline->Cutter->SetInputConnection(pipeline->ModelWarper->GetOutputPort());
 
+#if VTK_MAJOR_VERSION >= 9 || (VTK_MAJOR_VERSION >= 8 && VTK_MINOR_VERSION >= 2)
```

Comment:
I just realized that we should have cleanup the code in https://github.com/Slicer/Slicer/commit/1e4cc6293144ced711b625a912f472e59043b143. We will keep this in this topic and proceed to the cleanup after this is integrated.

Reply:
All of that said, the change makes  sense :+1: 

Cc: @lassoan